### PR TITLE
Update visualize.py

### PR DESCRIPTION
**Pull request by Yutong-gannis**
Add road details visualization

### CODE REVIEW

**Comment by Tumppi066** [comment]
These are changes that are definitely required.

Something I must make a note of though is that computing each of the lanes for the roads should be done in the roads.py file and then saved to the road class.

This pararrel road data is going to be required once I get to actually steering, so it's beneficial if it's computed beforehand. This also lower the fps drop for drawing each of the roads.

Also once I get the prefabs done I will be updating the road lane offsets based on the prefabs, and this is another one where I will be needing the pararrel lane data.

TLDR : Change the pararrel calculation to be done in the roads.py file, and saved to the road class. This way it's future proof and performance is improved.

**Comment by Yutong-gannis** [comment]
Ok

**Comment by Tumppi066** [comment]
Closed due to inactivity.
